mangaki/mangaki/models.py

```diff
@@ -122,6 +122,7 @@ class Work(models.Model):
     int_poster = models.FileField(upload_to='posters/', blank=True, null=True)
     nsfw = models.BooleanField(default=False)
     date = models.DateField(blank=True, null=True)
```

Comment:
Arguably, we should rename it into `start_date`, but, it may cost a lot to refactor this through the codebase, so let's jut put a comment on the field.